contrib/podmanimage/README.md

```diff
@@ -13,7 +13,7 @@ using the latest Fedora and then Podman is installed into them:
   * quay.io/podman/stable - This image is built using the latest stable version of Podman in a Fedora based container.  Built with podman/stable/Dockerfile.
   * quay.io/podman/upstream - This image is built using the latest code found in this GitHub repository.  When someone creates a commit and pushes it, the image is created.  Due to that the image changes frequently and is not guaranteed to be stable.  Built with podmanimage/upstream/Dockerfile.
   * quay.io/podman/testing - This image is built using the latest version of Podman that is or was in updates testing for Fedora.  At times this may be the same as the stable image.  This container image will primarily be used by the development teams for verification testing when a new package is created.  Built with podmanimage/testing/Dockerfile.
-
+  * quay.io/podman/stable:version - This image is built manually using a Fedora based container.  An RPM is first pulled from the [Fedora Updates System](https://bodhi.fedoraproject.org/) and the image is built from there.  For more details, see the Containerfile used to build it, podmanimage/stable/manual/Containerfile.
```

Comment:
Non blocking nit: I think we should link those Containerfiles to be easier accessible from the README. :)

Reply:
@TomSweeneyRedHat Fix the link then we can merge.

Reply:
oh excellent thought.  Incoming.